setup.py

```diff
@@ -4,6 +4,4 @@
 import setuptools
 
 
-setuptools.setup(
-    python_requires=">=3.7", setup_requires=["pbr>=2.0.0"], pbr=True
-)
+setuptools.setup(python_requires=">=3.7")
```

Comment:
See above, there is no need to remove the build dependency on pbr to solve the issue at hand.